### PR TITLE
Revision for bugs

### LINKED ISSUE #8: Uncertainty in request
What will a user get if he send a request to a person who is not registered? It will take a lot of time to wait an answer but nothing will happen.

### LINKED ISSUE #9: Database bug
If I'm not registered and I'm clicking on another button apart from "Register", then a bot stops to work.

### LINKED ISSUE #10: Bug with registration with EU phone number
If I register with foreign number(not russian) - a bot is not working for me.

### LINKED ISSUE #11: Too many buttons
Extremely large number of buttons and it's higly possible to misclick and select not what you want

### LINKED ISSUE #13: Add a state
"Track a person" needs it's own state as if someone misclicks and chooses "Track a person", it would be impossible to cancel it and bot will stop to work

### LINKED ISSUE #14: Give some buttons
When a user is being tracked and he has already sent his geoposition to all people eho tracked him, it's impossible to use a bot for him because no further actions are provided.

### PR DESCRIPTION
**Pull request by 7Dany6**
- [x] Solution for the problem concerning EU phone numbers
- [x] Solution for the problen concerning database bugs
- [x] Resolving problem about buttons galore
- [x] Providing clear solution for uncertainty in request
- [x] Create state for all functions
- [x] Send menu buttons for a tracked person

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 def unpack_geo(geopositions: list):
-    return '\n'.join(geopositions)
\ No newline at end of file
+    return '\n'.join(geopositions)
```
Comment (sulimovp) [inline functions.py:2]: add new line back

File: handlers.py [hunk 2]
```diff
@@ -61,40 +85,30 @@ async def process_feedback(message: types.Message, state: FSMContext):
...
-            await bot.send_message(chat_id=message.from_user.id, text="Thank you for the registration!")
-            await bot.send_message(USER_ID,
-                                   text=f"New user: {' '.join([message.from_user.first_name, f'@{message.from_user.username}', message.contact['phone_number']])}")
-        except sqlite3.IntegrityError:
-            await bot.send_message(chat_id=message.from_user.id, text="You've already been registered!")
-        await state.finish()
-
-
-@dp.message_handler(lambda message: message.text == "Track a person")
+@dp.message_handler(lambda message: message.text == "Track a person") # add state; index error if a user is not regitered
```
Comment (sulimovp) [inline handlers.py:88]: by PEP-8 you need to add 2 spaces between the code and "#" when making comment inside the line

File: functions.py (hunk 1 above)
Comment (7Dany6) [reply functions.py:2]: solved

File: handlers.py (hunk 2 above)
Comment (7Dany6) [reply handlers.py:88]: solved

**Pull request merged**